packages/xrpl/src/Wallet/defaultFaucets.ts

```diff
@@ -1,4 +1,5 @@
-import type { Client } from '..'
+import { Client } from 'xrpl-local'
```

Comment:
Why was this changed?

Reply:
Typescript complained about different definitions of types between `..` and `xrpl-local`. It considered the types as different types altogether leading to type mismatches.

Reply:
Why does `xrpl-local` exist here? I thought it was just a placeholder for tests and only existed in the test scope.

Reply:
Fixed